toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+set -e
+
+script_dir=$(realpath "$(dirname "$0")")
+topdir=/usr/src/mariner
+enable_local_repo=false
+
+switch_to_red_text() {
+    printf "\e[31m"
+}
+
+switch_to_normal_text() {
+    printf "\e[0m"
+}
+
+print_error() {
+    echo ""
+    switch_to_red_text
+    echo ">>>> ERROR: $1"
+    switch_to_normal_text
+    echo ""
+}
+
+help() {
+echo "
+Usage:
+sudo make containerized-rpmbuild [REPO_PATH=/path/to/CBL-Mariner] [MODE=test|build] [VERSION=1.0|2.0] [MOUNTS= /src/path:/dst/path] [ENABLE_REPO=y]
+
+Starts a docker container with the specified version of mariner.
+
+Optional arguments:
+    REPO_PATH:      path to the CBL-Mariner repo root directory. default: "current directory"
+    MODE            build or test. default:"build"
+                        In 'test' mode it will use a pre-built mariner chroot image.
+                        In 'build' mode it will use the latest published container.
+    VERISION        1.0 or 2.0. default: "2.0"
+    MOUNTS          mount a directory into the container. Should be of the form '/src/dir:/dest/dir'. 
+                    Can be specified multiple times.
+    ENABLE_REPO:    Set to 'y' to use local RPMs to satisfy package dependencies. default: "n"
+
+To see help, run 'sudo make containerized-rpmbuild-help'
+"
+}
+
+build_chroot() {
```

Comment:
Some of these function names seem odd. `build_chroot()` runs `make graph-cache`. And `build_graph()` runs `sudo make workplan`.

Reply:
I have named them according to what they're doing/ meant for. I'm not sure how the toolkit make commands have been named. Agreed about the oddity here